engine/src/test/scala/cromwell/engine/WorkflowDescriptorSpec.scala

```diff
@@ -171,7 +171,9 @@ class WorkflowDescriptorSpec extends FlatSpec with Matchers with ScalaFutures {
           1,
           Map.empty,
           None,
-          None)
+          None,
+          false,
+          false)
         call -> Seq(metadata)
     }).toMap
 
```

Comment:
Yeah, there isn't.  I'm actually struggling to find where our metadata endpoint tests are in general...  Do you know where they are?